nx.json

```diff
@@ -33,5 +33,15 @@
     "version": {
       "preVersionCommand": "yarn nx run-many -t build"
     }
+  },
+  "targetDefaults": {
+    "test": {
+      "dependsOn": [
+        {
+          "projects": ["api"],
+          "target": "build-workers"
+        }
+      ]
+    }
```

Comment:
We can remove this configuration for now and decide whether we should add it later.